### PR TITLE
timeshift-19.01.ebuild (New Package)

### PR DESCRIPTION
**Pull request by lambroisie**
Hi!

Please find attached timeshift-19.01.ebuild. Uses 'sed' to add `AppExcludeEntry' to the root namespace.

Timeshift for Linux is an application that provides functionality similar to the System Restore feature in Windows and the Time Machine tool in macOS. Timeshift protects your system by taking incremental snapshots of the file system at regular intervals. These snapshots can be restored at a later date to undo all changes to the system.

I suggest app-backup/timeshift

https://bugs.gentoo.org/690258

### CODE REVIEW

File: app-backup/timeshift/metadata.xml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<maintainer type="person">
+		<email>christinavitting@gmail.com</email>
+		<name>Christina Vitting</name>
```
Comment (hydrapolic) [inline app-backup/timeshift/metadata.xml]: Please add proxy-maint.

File: app-backup/timeshift/timeshift-19.01.ebuild [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+DEPEND="
+	>=x11-libs/gtk+-3.24.8
+	>=net-libs/libsoup-2.64.2
+	dev-util/desktop-file-utils
+	net-misc/rsync
+	dev-libs/json-glib
+	sys-process/cronie
+	>=x11-libs/xapps-1.4.2
+	>=dev-libs/libgee-0.20.1
```
Comment (hydrapolic) [inline app-backup/timeshift/timeshift-19.01.ebuild]: Please sort the deps for easier readability

File: app-backup/timeshift/timeshift-19.01.ebuild [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+src_prepare() {
+	sed -e 's:--thread:& Core/AppExcludeEntry.vala:' -i 'src/makefile'
+	mkdir ${S}/tmpbin
+	ln -s $(echo $(whereis valac-) | grep -oE "[^[[:space:]]*$") ${S}/tmpbin/valac
+	vala_src_prepare
+	default
+}
+
+src_configure() {
+	PATH="${S}/tmpbin/:$PATH"
```
Comment (hydrapolic) [inline app-backup/timeshift/timeshift-19.01.ebuild]: Why are you doing the magic with valac? Cannot it find it in his normal system location?

**Comment by Simbaclaws** [comment]
Is there anyone that can check whether this ebuild is current and whether it can be merged with gentoo?
I would love being able to use timeshift for my daily backups :+1:

**Pull request closed without merging**